code/__HELPERS/_macros.dm

```diff
@@ -5,8 +5,30 @@
 // fun if you want to typecast humans/monkeys/etc without writing long path-filled lines.
 #define ishuman(A) istype(A, /mob/living/carbon/human)
 
+#define isjusthuman(A) A.type == /mob/living/carbon/human
+
 #define ismonkey(A) istype(A, /mob/living/carbon/monkey)
 
+#define isvox(A) istype(A, /mob/living/carbon/human/vox)
```

Comment:
Would I need to cast A as a var/mob/living/carbon/human first in order to access its species var?